STANDARD.md

```diff
@@ -438,22 +438,22 @@ short-hand notation (see: *Constant Record Components*).
                               with `constant record components`
     - example: reading example (with h5py) in Python:
 ```python
-def is_const_record(record_name, component_name) :
-    return ("value" in record_name["component_name"].attrs.keys())
+def is_const_record(record, component_name) :
```

Comment:
this was just readability: both are suffixed `_name`, but the record was actually the hdf5 object, but `component_name` is a string.